server/routes/liability.js

```diff
@@ -1,10 +1,55 @@
 const router = require('express').Router();
-const { Liability } = require('../db');
+const { Liability, Device, User } = require('../db').models;
 
-function check(req, res) {
-  res.send('It works!!');
+const DEFAULT_LIMIT = 30;
+
+function create(req, res, next) {
+  return Liability.create(req.body)
+    .then(liability => res.send(liability))
+    .catch(err => next(err));
+}
+
+function get(req, res, next) {
+  const where = { id: req.params.id };
+  const include = [{ model: Device }, { model: User }];
+
+  return Liability.find({ where, include })
```

Comment:
/rant
I'm not aware of `Model.find` method in sequelize? :confused:
Ofc I'm not aware, that thing is magical alias for `Model.findOne` which is mentioned in f*** source code **only**: https://github.com/sequelize/sequelize/blob/v4.13.8/lib/model.js#L1684

Leaving that aside, you could use `Model.findById` here by doing:
```js
Liability.findById(req.params.id, { where, include });
```
which is something you probably don't know because it is written **only** [here](http://docs.sequelizejs.com/class/lib/model.js~Model.html#static-method-findById), at the bottom, ofc :rage:

Reply:
For the `.find` it's actually also stated [here](http://docs.sequelizejs.com/class/lib/model.js~Model.html#static-method-findOne). 😉 
Are you sure about `.findById`? Its options are specified to have only `transaction` and `searchPath` props... [Take a look](http://docs.sequelizejs.com/class/lib/model.js~Model.html#static-method-findById)